package.json

```diff
@@ -26,7 +26,7 @@
   },
   "dependencies": {
     "promise-pjs": "1.1.3",
-    "web-activities": "1.4.0"
+    "web-activities": "1.4.0",
```

Comment:
I don't think you are allowed terminating commas in package.json.

Reply:
Done.